ranger/core/fm.py

```diff
@@ -22,11 +22,7 @@
 from ranger.gui.ui import UI
 from ranger.container.bookmarks import Bookmarks
 from ranger.core.runner import Runner
-from ranger.ext.img_display import (W3MImageDisplayer, ITerm2ImageDisplayer,
-                                    TerminologyImageDisplayer,
-                                    URXVTImageDisplayer, URXVTImageFSDisplayer,
-                                    KittyImageDisplayer, UeberzugImageDisplayer,
-                                    ImageDisplayer)
+from ranger.ext import img_display
```

Comment:
Since we only use the registry, can we import this as `from ranger.ext.img_display import IMAGE_DISPLAYER_REGISTRY as imgRegistry` or something?

Reply:
Sure. I can do that. Would it be at all preferable to just expose a `ranger.ext.img_display.get_image_displayer` and replace the uses of `FM._get_image_displayer` with it?

Reply:
Or go full ham and replace image_displayer with a data descriptor?

Reply:
Sounds like a good suggestion, re the first. Re the second, depends on what exactly you call a data descriptor. I'm not well-versed in OOP design patterns.